mock/src/main/java/jinsist/mock/MethodToStubNotFound.java

```diff
@@ -1,6 +1,8 @@
 package jinsist.mock;
 
 class MethodToStubNotFound extends RuntimeException {
+    private static final long serialVersionUID = 1L;
```

Comment:
I see in Eclipse's Problems view. `javac` shows warnings with `-Xlint` parameter.